install/primehub-prerequisite

```diff
@@ -32,17 +32,16 @@ error() {
 usage (){
     local SELF=`basename $0`
     cat <<EOF
+
+PURPOSE:
+  Check the following  1) linux_package 2) container 3) Kubernetes 4) ingress 5) gpu 6) storageclass
+
 USAGE:
-  $SELF check linux_package    : check the Linux Package
-  $SELF check container        : check the Container runtime
-  $SELF check kubernetes       : check the Kubernetes
-  $SELF check ingress          : check Ingress
-  $SELF check gpu              : check GPU
-  $SELF check storageclass     : check StorageClass
-
-  $SELF install linux_package  : check the Linux Package
   $SELF -h,--help              : show this message
 
+  $SELF install : check and install prerequisite if missing
+  $SELF check   : check only
+
```

Comment:
@jimmyliao 
Please help to write it clearly
```
PURPOSE:
  Check the following pre-requisite demand:
  
  Command line:
  $SELF install <prerequisted-demand>: check and install prerequisite if missing
  $SELF check <prerequisted-demand>: check only
  
 prerequisted-demand Option:
  (1) linux_package 
  (2) container 
  (3) Kubernetes 
  (4) ingress 
  (5) gpu 
  (6) storageclass
```

Reply:
> @jimmyliao Please help to write it clearly
> 
> ```
> PURPOSE:
>   Check the following pre-requisite demand:
>   
>   Command line:
>   $SELF install <prerequisted-demand>: check and install prerequisite if missing
>   $SELF check <prerequisted-demand>: check only
>   
>  prerequisted-demand Option:
>   (1) linux_package 
>   (2) container 
>   (3) Kubernetes 
>   (4) ingress 
>   (5) gpu 
>   (6) storageclass
> ```

above Option: will confuse user to have more advance option to select

Reply:
The updated prerequisite is simply two part: check and install, the install will check all and bypass and continue next section if full-filled.

Reply:
please play around on your machine which not install any lib or partly installed. Thanks

Reply:
@jimmyliao Got it. I just make some mistake here. I will merge the request.